templates/zerver/api/roles-and-permissions.md

```diff
@@ -0,0 +1,85 @@
+# Roles and permissions
+
+Zulip offers [several levels of permissions based on user
+organization-level roles](/help/roles-and-permissions).
+
+Here are some important details to note when working with these
+roles and permissions in Zulip's API:
+
+## A user's organization-level role
+
+As of [Zulip 4.0](/api/changelog#changes-in-zulip-40) (feature levels
+59 and 60), endpoints and [events](/api/get-events#realm_user-add)
+that return user account data include a `role` property, which
+contains the user's organization-level role. These roles are encoded as:
+
+* Organization owner: 100
+
+* Organization administrator: 200
+
+* Organization moderator: 300
+
+* Member: 400
+
+* Guest: 600
+
+User account data also include these relevant boolean properties:
```

Comment:
I think it's worth explicitly mentioning that all of these except for `is_billing_admin` duplicate of the `role` field, present for help readability in simpler clients.  (Which might be best done by talking about `is_billing_admin` separately).